### ISSUE TITLE
Call to undefined method DataProviderTestSuite::usesDataProvider()

### ISSUE THREAD

**Issue by kdambekalns** [sebastianbergmann/phpunit#3441]
| Q                   | A
| --------------------| ---------------
| PHPUnit version     | 7.5.0
| PHP version         | 7.1.11
| Installation Method | Composer

As can be seen at https://travis-ci.org/neos/flow-development-collection/jobs/464914582 I get

```
PHP Fatal error:  Uncaught Error: Call to undefined method PHPUnit\Framework\DataProviderTestSuite::usesDataProvider() in /home/travis/build/neos/flow-development-distribution/Packages/Libraries/phpunit/phpunit/src/Util/Log/JUnit.php:297
Stack trace:
#0 /home/travis/build/neos/flow-development-distribution/Packages/Libraries/phpunit/phpunit/src/Framework/TestResult.php(400): PHPUnit\Util\Log\JUnit->startTest(Object(PHPUnit\Framework\DataProviderTestSuite))
#1 /home/travis/build/neos/flow-development-distribution/Packages/Libraries/phpunit/phpunit/src/Framework/TestSuite.php(708): PHPUnit\Framework\TestResult->startTest(Object(PHPUnit\Framework\DataProviderTestSuite))
#2 /home/travis/build/neos/flow-development-distribution/Packages/Libraries/phpunit/phpunit/src/Framework/TestSuite.php(746): PHPUnit\Framework\TestSuite->run(Object(PHPUnit\Framework\TestResult))
#3 /home/travis/build/neos/flow-development-distribution/Packages/Libraries/phpunit/phpunit/src/TextUI/TestRunner.php(629): PHPUnit\Framework\TestSuite->run(Object(PHPUn in /home/travis/build/neos/flow-development-distribution/Packages/Libraries/phpunit/phpunit/src/Util/Log/JUnit.php on line 297
```

when running this on Travis. It works fine locally. Anyone else?

**Comment by epdenouden** [comment]
@kdambekalns thanks for bug report. I recently did some where in that early that got merged into 7.5, which didn't cause any errors in our own collection of tests.

I will investigate why this breaks in Travis and have a fix tomorrow.

**Comment by epdenouden** [comment]
@kdambekalns A fix has been merged into 7.5.1 by Sebastian. Let me know whether this fixes your CI troubles.

**Comment by epdenouden** [comment]
@sebastianbergmann this ticket can be closed

**Comment by kdambekalns** [comment]
Thanks for fixing! 👍

But, no 7.5.1 released yet, so Travis still pulls in 7.5.0 for now. Will check again after release…

**Comment by kdambekalns** [comment]
Works fine now, thanks!
